<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>optionsstartatboot</class>
 <widget class="QDialog" name="optionsstartatboot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>79</x>
     <y>10</y>
     <width>331</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
</string>
   </property>
   <property name="text">
    <string>STARTUP AT BOOT  OPTIONS</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>230</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>320</y>
     <width>230</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Startatboot_comboBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>OFF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ON</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>100</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>START MODE AT BOOT</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>ROVER MODE AT BOOT</string>
   </property>
  </widget>
  <widget class="QComboBox" name="RoverMode_comboBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SINGLE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>SBAS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DGPS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PPP STATIC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RTK STATIC</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RTK KINE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CUSTOM 1</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>CHOOSE MODE AT BOOT</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ChooseMode_comboBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>140</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ROVER</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BASE</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
